test/sql-feature-flags/mzcompose.py

```diff
@@ -226,35 +226,6 @@ def query_item(cls, ordinal: int) -> str:
         assert False, "query_item() must be overriden"
 
 
-class WithMutuallyRecursive(FeatureTestScenario):
-    @classmethod
-    def feature_name(cls) -> str:
-        return "enable_with_mutually_recursive"
-
-    @classmethod
-    def feature_error(cls) -> str:
-        return "WITH MUTUALLY RECURSIVE is not supported"
-
-    @classmethod
-    def create_item(cls, ordinal: int) -> str:
-        return dedent(
-            f"""
-            CREATE VIEW wmr_{ordinal:02d} AS WITH MUTUALLY RECURSIVE
-                foo (a int, b int) AS (SELECT 1, 2 UNION SELECT a, 7 FROM bar),
-                bar (a int) as (SELECT a FROM foo)
-            SELECT * FROM bar;
-            """
-        )
-
-    @classmethod
-    def drop_item(cls, ordinal: int) -> str:
-        return f"DROP VIEW wmr_{ordinal:02d}"
-
-    @classmethod
-    def query_item(cls, ordinal: int) -> str:
-        return f"SELECT * FROM wmr_{ordinal:02d}"
-
-
```

Comment:
If it doesn't exist already, we might want to change this into a platform check.

Reply:
Now that there isn't a feature flag, this isn't really testing anything other than that a simple WMR view (the `foo` above) can be created, queried, and dropped without things breaking. I am sure we already have test coverage for such a case.